### PR TITLE
Adding menu options in sidebar, and improving order

### PR DESCRIPTION
**Pull request by asaadmahmood**
#### Summary
- Adding menu options in sidebar, and improving order of various menus
- Improving styling for the modals as well

Ticket:
https://community.mattermost.com/boards/team/qghzt68dq7bopgqamcnziq69ao/b4ctmm917q3rzdfi451qtuxjzcw/vuswsftt1rb873datosmmbtwkpa/c66rwa7zc33ghb8rfw7gixcqo8c

#### Screenshot
Screenshot 2022-08-20 at 12 14 05 AM
Screenshot 2022-08-20 at 12 15 37 AM
Screenshot 2022-08-20 at 12 23 49 AM

### CODE REVIEW

**Review by Rajat-Dabade — COMMENTED** [review]
@asaadmahmood, Git CI is failing for not updating the snapshot for your changes.

**Comment by Rajat-Dabade** [comment]
Removed the label `CherryPick/Approved` as there is no release branch for `7.3`

File: webapp/src/components/boardsSwitcher/boardsSwitcher.tsx [hunk 1]
```diff
@@ -93,20 +109,70 @@ const BoardsSwitcher = (props: Props): JSX.Element => {
...
             }
 
             {
                 showSwitcher &&
                 <BoardSwitcherDialog onClose={() => setShowSwitcher(false)} />
             }
+
+            {
+                showCreateCategoryModal && (
+                    <CreateCategory
```
Comment (harshilsharma63) [inline webapp/src/components/boardsSwitcher/boardsSwitcher.tsx:150]: This is duplicated code. Can you try to move the logic inside the component itself? I can help with this.

**Review by harshilsharma63 — CHANGES_REQUESTED** [review]

**Comment by harshilsharma63** [comment]
FOund one bug - previously the option to create a new category shows up on the default "Boards" category as well. Now it doesn't. All it shows is an empty menu.

**Comment by asaadmahmood** [comment]
Bug fixed.

File: webapp/src/components/boardsSwitcher/boardsSwitcher.tsx (hunk 1 above)
Comment (asaadmahmood) [reply webapp/src/components/boardsSwitcher/boardsSwitcher.tsx:150]: Sure.

**Review by harshilsharma63 — APPROVED** [review]
Other than the linter errors, LGTM 👍

**Pull request merged**